pkgs/build-support/bintools-wrapper/add-hardening.sh

```diff
@@ -27,32 +28,31 @@ if (( "${NIX_DEBUG:-0}" >= 1 )); then
   printf 'HARDENING: disabled flags:' >&2
   (( "${#hardeningDisableMap[@]}" )) && printf ' %q' "${!hardeningDisableMap[@]}" >&2
   echo >&2
-fi
 
-if (( "${#hardeningEnableMap[@]}" )); then
-  if (( "${NIX_DEBUG:-0}" >= 1 )); then
+  if (( "${#hardeningEnableMap[@]}" )); then
     echo 'HARDENING: Is active (not completely disabled with "all" flag)' >&2;
```

Comment:
@Ericson2314 Did you mean to drop the `if (( "${NIX_DEBUG:-0}" >= 1 ))` condition? We'll start getting a lot more noise in our builds, and I'd fear some builds failing due to the extra (unexpected) output.

Reply:
Oh wait, never mind. I see. This is tucked under a similar condition anyway. I wasn't following the diff closely enough. :+1: